doc/analytics.rst

```diff
@@ -8,16 +8,13 @@ Puoi trovare questi dati negli “Insight” su Facebook e negli “Analytics”
 
 .. admonition:: DEVI
 
-   Monitorare con continuità i dati delle pagine social per capire come indirizzare la programmazione. Condividere i risultati numerici con    il gruppo di lavoro per attuare azioni correttive, coerenti con la strategia di comunicazione complessiva. 
+   Monitorare con continuità i dati delle pagine social per capire come indirizzare la programmazione. Condividere i risultati numerici    con il gruppo di lavoro per attuare azioni correttive, coerenti con la strategia di comunicazione complessiva. 
```

Comment:
@martafalzone qui, e in altri punti, ci sono diversi spazi tra le parole. Non impattano sulla resa del documento, ma diminuiscono la leggibilità dei sorgenti.

Reply:
ok grazie Davide! modifiche apportate al fork per integrazione.